cmd/keytransparency-sequencer/server.go

```diff
@@ -18,24 +18,26 @@ import (
 	"flag"
 	"net/http"
 
+	"github.com/google/keytransparency/cmd/serverutil"
+
 	"github.com/golang/glog"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/reflection"
 
-	"github.com/google/keytransparency/cmd/serverutil"
-	ktpb "github.com/google/keytransparency/core/proto/keytransparency_v1_grpc"
+	gpb "github.com/google/keytransparency/core/proto/keytransparency_v1_grpc"
+	_ "github.com/google/trillian/crypto/keys/der/proto"
 	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
 )
 
 var (
-	addr     = flag.String("metrics-addr", ":8081", "The ip:port to publish metrics on")
+	addr     = flag.String("addr", ":8080", "The ip:port to serve on")
```

Comment:
not sure why the default number changed?

Reply:
Previously, metrics was an http service that just served metrics. 
Now, it's an https service that is a proper gRPC and HTTPS / JSON server.   The number change is for consistency with the other Key Transparency binaries.